google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/AbstractTestEnv.java

```diff
@@ -101,17 +101,17 @@ public boolean shouldWaitForCmekKeyStatusUpdate() {
   }
 
   public String getPrimaryZone() {
-    return "us-central1-b";
-  }
-
-  public String getPrimaryRegionSecondZone() {
-    return "us-central1-c";
+    return "us-east1-c";
```

Comment:
Yea we could do that. I did run tests locally and the other tests weren't affected so felt confident changing this. Either way, it would be good to test with the us-east1 regions as it has 2 zones we can test with - so would be good if you're able to add that key. 
Maybe I can do the dynamic zone determination as another PR so we can unblock our internal stuff?

Reply:
Okay, sounds good. I'll just hard-code change to us-east1 zones 👍